python/ray/serve/replica.py

```diff
@@ -414,9 +414,18 @@ async def invoke_single(self, request_item: Query) -> Tuple[Any, bool]:
             if len(inspect.signature(runner_method).parameters) > 0:
                 result = await method_to_call(*args, **kwargs)
             else:
-                # The method doesn't take in anything, including the request
-                # information, so we pass nothing into it
-                result = await method_to_call()
+                # When access via http http_arg_is_pickled with no args:
+                # args = (<starlette.requests.Request object at 0x7fe900694cc0>,)
```

Comment:
cc: @simon-mo i read the code that generates this a bit but didn't quite understand what's going on ... why do we need to wrap a new request object here if input sig is empty ?